### ISSUE TITLE
Segmentation fault when using --dims

### ISSUE THREAD

**Issue by flemmens** [hobuinc/untwine#150]
Hello,

When I am trying to use the --dims option, I receive a "Segmentation fault" error.

This works fine:
```
untwine --files=mydata.las --single_file=true --output_dir=out
```

This does not work (Segmentation fault):
```
untwine --files=mydata.las --single_file=true --dims="X, Y, Z" --output_dir=out
```

Is this the correct way to use the --dims option ? There is not much info in the README. I would like to keep only X,Y,Z values.

**Comment by abellgithub** [comment]
"output_dir" is for multi-file output. Just specify the output file as an argument.

I'll try to harden the argument handling.

**Comment by abellgithub** [comment]
I forgot that this was correct syntax for the filename. I can't reproduce, so if you have a file that causes this behavior and can post, I can take a look.

**Comment by abellgithub** [comment]
Reproduced.

**Comment by abellgithub** [comment]
Fixed by #151
